Tests/PerformanceTests.swift

```diff
@@ -0,0 +1,124 @@
+// Created by Cal Stephens on 2/14/22.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import Foundation
+import XCTest
+
+@testable import Lottie
+
+// MARK: - PerformanceTests
+
+final class PerformanceTests: XCTestCase {
+
+  // MARK: Internal
+
+  func testAnimationViewSetup_simpleAnimation() {
+    // Compare the performance of displaying this simple animation in the two animation engines
+    let ratio = compareEngineSetupPerformance(for: simpleAnimation, iterations: 2000)
+
+    // This is basically a snapshot test for the performance of the Core Animation engine
+    // compared to the Main Thread engine. Currently, the Core Animation engine is
+    // about 1.6x slower than the Main Thread engine in this example.
+    XCTAssertEqual(ratio, 1.6, accuracy: 0.2)
+  }
+
+  func testAnimationViewSetup_complexAnimation() {
+    let ratio = compareEngineSetupPerformance(for: complexAnimation, iterations: 500)
+    XCTAssertEqual(ratio, 1.75, accuracy: 0.3)
+  }
+
+  func testAnimationViewScrubbing_simpleAnimation() {
+    let ratio = compareEngineScrubbingPerformance(for: simpleAnimation, iterations: 1000)
+    XCTAssertEqual(ratio, 0.05, accuracy: 0.02)
+  }
+
+  func testAnimationViewScrubbing_complexAnimation() {
+    let ratio = compareEngineScrubbingPerformance(for: complexAnimation, iterations: 1000)
+    XCTAssertEqual(ratio, 0.002, accuracy: 0.001)
```

Comment:
I was pretty impressed with this number -- interactively scrubbing the current frame of a complex animation in the new rendering engine is up to 500x faster than in the main thread engine

Reply:
(not too surprising, since to scrub the Core Animation engine you just have to update the `timeOffset` of the root layer)